src/stactools/ephemeral/stac.py

```diff
@@ -1,117 +1,67 @@
-import logging
 from datetime import datetime, timezone
 
+import stactools.core.create
 from pystac import (
-    Asset,
-    CatalogType,
     Collection,
     Extent,
     Item,
-    MediaType,
-    Provider,
-    ProviderRole,
     SpatialExtent,
     TemporalExtent,
 )
-from pystac.extensions.projection import ProjectionExtension
-
-logger = logging.getLogger(__name__)
 
 
 def create_collection() -> Collection:
-    """Create a STAC Collection
-
-    This function includes logic to extract all relevant metadata from
-    an asset describing the STAC collection and/or metadata coded into an
-    accompanying constants.py file.
+    """Creates a STAC Collection.
 
-    See `Collection<https://pystac.readthedocs.io/en/latest/api.html#collection>`_.
+    This function should create a collection for this dataset. See `the STAC
+    specification
+    <https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md>`_
+    for information about collection fields, and
+    `Collection<https://pystac.readthedocs.io/en/latest/api.html#collection>`_
+    for information about the PySTAC class.
 
     Returns:
         Collection: STAC Collection object
     """
-    providers = [
-        Provider(
-            name="The OS Community",
-            roles=[ProviderRole.PRODUCER, ProviderRole.PROCESSOR, ProviderRole.HOST],
-            url="https://github.com/stac-utils/stactools",
-        )
-    ]
-
-    # Time must be in UTC
-    demo_time = datetime.now(tz=timezone.utc)
-
     extent = Extent(
         SpatialExtent([[-180.0, 90.0, 180.0, -90.0]]),
-        TemporalExtent([[demo_time, None]]),
+        TemporalExtent([[datetime.now(tz=timezone.utc), None]]),
     )
 
     collection = Collection(
-        id="my-collection-id",
-        title="A dummy STAC Collection",
-        description="Used for demonstration purposes",
-        license="CC-0",
-        providers=providers,
+        id="example-collection",
+        title="Example collection",
+        description="An example collection",
         extent=extent,
-        catalog_type=CatalogType.RELATIVE_PUBLISHED,
+        extra_fields={"custom_attribute": "foo"},
     )
-
     return collection
 
 
 def create_item(asset_href: str) -> Item:
-    """Create a STAC Item
+    """Creates a STAC item from a raster asset.
+
+    This example function uses :py:func:`stactools.core.utils.create_item` to
+    generate an example item.  Datasets should customize the item with
+    dataset-specific information, e.g.  extracted from metadata files.
 
-    This function should include logic to extract all relevant metadata from an
-    asset, metadata asset, and/or a constants.py file.
+    See `the STAC specification
+    <https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md>`_
+    for information about an item's fields, and
+    `Item<https://pystac.readthedocs.io/en/latest/api.html#item>`_ for
```

Comment:
Missing an `s` for the anchor link.
`https://pystac.readthedocs.io/en/latest/api.html#items` or `https://pystac.readthedocs.io/en/latest/api/pystac.html#pystac.Item`